index.html

```diff
@@ -686,16 +679,8 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
  <a>Timing-Allow-Origin</a> header fields by appending each
  subsequent field value to the combined field value in order,
  separated by a comma.</p>
- <p>The <dfn>timing allow check</dfn> algorithm, which checks
-whether a resource's timing information can be shared with the
- <a>current document</a>, is as follows:</p>
- <ol>
-     <li>Let <var>response</var> be the resource's <a data-cite=
-       "FETCH#concept-response">Response</a>.</li>
-     <li>Return <var>response</var>'s <a data-cite=
-       "FETCH#concept-response-timing-allow-passed">
-       timing allow passed flag</a>.</li>
- </ol>
+ <p>The <a>Timing-Allow-Origin</a> check is performed as part of
```

Comment:
TAO is a header, not a check, so you may want to word this more precisely (the headers are processed in fetch to compute the attributes accordingly?)

Reply:
fixed